### PR TITLE
Addressing workshop feedback

### PR DESCRIPTION
**Pull request by artberger**
Addresses issue called out in slack channel: https://ibm-argonauts.slack.com/messages/CBKP5QLN9

Addresses #47, #48

### CODE REVIEW

File: workshop/deploy_rollup.sh [hunk 1]
```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 echo "Install IBM Cloud CLI"
-. workshop/install_bx.sh
+. workshop/install_ibmcloud.sh
```
Comment (jzaccone) [inline workshop/deploy_rollup.sh]: Need to change filename to match

**Comment by MHBauer** [comment]
@artberger what was feedback? I can't see that slack.

File: workshop/Lab3/README.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ try to add, or remove, pods from the system to match your request. To can
 make these modifications by using the following command:
 
    ```console
-   $ kubectl edit deployment guestbook
+   $ kubectl edit deployment guestbook-v1
```
Comment (MHBauer) [inline workshop/Lab3/README.md]: why would the deployment be called this?

Comment (MHBauer) [reply workshop/Lab3/README.md]: I see, guestbook has been updated. We need a better way to keep this in sync if we're going to have separate repos. Checking out a specific revision/branch or something.

**Comment by artberger** [comment]
@MHBauer this commit addressed the feedback in slack: https://github.com/IBM/kube101/pull/50/commits/ed9df25c482bc7501a1e582e9c4596bf727a9ae2, besides the plugin name change and no need for the alias.

File: workshop/Lab3/README.md (hunk 2 above)
Comment (artberger) [reply workshop/Lab3/README.md]: Yeah it had to do with the tag. Is there something you want me to change in this PR?

**Comment by duglin** [comment]
@brahmaroutu please review

**Comment by MHBauer** [comment]
we need to rebase and merge this. I don't have perms to do either.

**Comment by MHBauer** [comment]
I have some perms now, maybe @jzaccone can follow up.

**Comment by stevemar** [comment]
Closing due to inactivity and conflicts

**Pull request closed without merging**